packages/common/src/TimeUtils.js

```diff
@@ -14,6 +16,16 @@ const averageBlockTimeSeconds = async (/* lookbackSeconds */) => {
   }
 };
 
+// Sets fromBlock to the value of an environment variable if one is set. This can be set to 0 to make tests work with Ganache, or any other value needed for a production script or bot.
+async function getFromBlock() {
+  if (process.env.FROM_BLOCK) {
+    return process.env.FROM_BLOCK;
+  } else {
+    return UMA_FIRST_EMP_BLOCK;
+  }
+}
+
 module.exports = {
-  averageBlockTimeSeconds
+  averageBlockTimeSeconds,
+  getFromBlock
```

Comment:
Not strongly opinionated either way, but would this be better suited to sit inside financial-templates-lib under helpers or even the ExpiringMultiPartyClient, itself? I only ask because I think we should generally try to remove things from common unless they are common functionality used by multiple packages.

Reply:
I think `getFromBlock` would be useful in other packages when you want to set a custom `fromBlock` but don't want to hardcode it. Affiliate scripts and bots are possible examples.

Reply:
I could see a use for this in the affiliates package, the one gripe I have though is having to set a new env var to the run tests outside of main net.

Reply:
yeah and we originally used this function in the affiliates package for calculating Gas fee rebates